public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -228,6 +228,15 @@ export class DashboardScenePageStateManager extends StateManagerBase<DashboardSc
     return null;
   }
 
+  public async updateNavModel(folderUid: string) {
```

Comment:
What do you think about updating signature for [`initNavModel`](https://github.com/grafana/grafana/pull/86342/files#diff-014099493438b3f54a14bf4d12e496c65d45fc3dd9b8a91f27a0d1571f538015L231) to keep consistent the function signatures?

Also, I'd add some tests here. WDYT?

Reply:
In that case we might as well cut one of the functions and use the same everywhere, which is what I'm going for now :D